### PR TITLE
Update docblocks.

### PR DESCRIPTION
**Pull request by ADmad**

### CODE REVIEW

File: src/View/View.php [hunk 1]
```diff
@@ -497,7 +497,7 @@ public function autoLayout($autoLayout = null)
     /**
      * Get the current view theme.
      *
-     * @return string
+     * @return string|null
```
Comment (dereuromark) [inline src/View/View.php]: Can the setters then also accept null? E.g. `setTheme($theme)`

Comment (ADmad) [reply src/View/View.php]: Yes it can.

Comment (dereuromark) [reply src/View/View.php]: We should also point that out. One more advantage of separating the concerns in separate methods :)

Comment (ADmad) [reply src/View/View.php]: I updated docblock of `setTheme()` too.

**Comment by ADmad** [comment]
I also noticed the `set*` methods of `View` class don't return `$this` like other setters throughout the framework. Should I update those return types?

**Comment by markstory** [comment]
@ADmad Yes.

**Pull request merged**